search/lib/convert/granules.js

```diff
@@ -117,8 +119,8 @@ function cmrGranToFeatureGeoJSON (event, cmrGran) {
       {
         rel: 'root',
         href: generateAppUrl(event)
-      },
-      wfs.createLink('metadata', cmr.makeCmrSearchUrl(`/concepts/${cmrGran.id}.native`))
+      }
+      // wfs.createLink('metadata', cmr.makeCmrSearchUrl(`/concepts/${cmrGran.id}.native`))
```

Comment:
before we merge to master, we should remove these commented-out lines.

Reply:
Good catch!